sonar-plugin/src/main/java/io/ecocode/javascript/CheckList.java

```diff
@@ -43,7 +43,8 @@ public static List<Class<? extends JavaScriptCheck>> getAllChecks() {
                 NoMultipleStyleChanges.class,
                 PreferCollectionsWithPagination.class,
                 PreferShorthandCSSNotations.class,
-                ProvidePrintCSS.class
+                ProvidePrintCSS.class,
+                AvoidAutoPlay.class
```

Comment:
Can you put it in an alphabetical order? This will prevent (I hope) some conflicts with other PRs